softlayer/cloud-config.yml

```diff
@@ -0,0 +1,51 @@
+azs:
+- name: z1
+  cloud_properties:
+    datacenter: { name: ((sl_datacenter)) }
+
+vm_types:
+- name: default
+  cloud_properties:
+    bosh_ip: ((internal_ip))
```

Comment:
It's used to combine mbus url in user_data.json, we are already able to set the private IP for dynamic networking, but haven't implemented it for static networking.